### ISSUE TITLE
Twitch Recover downloads to Appdata/Local/Temp First

### ISSUE THREAD

**Issue by Surg-Dev** [TwitchRecover/TwitchRecover#119]
This is not a major bug, per se, however, even when given a directory on a separate drive, Twitch Recover downloads to Appdata/Local/Temp, which my C drive does not have room for. Possibly add a feature to download it entirely in one location?

**Comment by dsnvwlmnt** [comment]
Yup. It's actually a significant issue, because there's no indication anywhere of this going on. And especially in combination with other bugs. My C:\was getting eaten up and I couldn't figure out why until I found out about these temp files.

Been using this program for just 5 days, and yet check out what I found in the image below: - 22 million, yes 22 million temp files in there, totalling 28 GB. The program seems to create 1 folder per CMD instance you run, and dumps everything into that one folder. If your download hangs forever, it will download phantom 0 KB files forever (that's what was going on in the first 6 folder example of 11M files).

![image](https://user-images.githubusercontent.com/14173022/133692618-5b616a2d-b6e8-482d-9ee6-d11995195152.png)
